record.go

```diff
@@ -75,7 +76,7 @@ func Record(cmd *cobra.Command, args []string) error {
 	}
 
 	if err := pty.InheritSize(os.Stdin, terminal); err != nil {
-		log.Printf("error resizing pty: %s", err)
+		logger.Printf("error resizing pty: %s", err)
```

Comment:
This is probably ok to be ignored for the `--quiet` flag, but would be helpful for the `--debug` mode

Reply:
Yeah, I agree with this point.
Maybe we could have a better description about the logger modes and continue improving this point in future pr's.